test/testinput/bufr_ncep_highRes_sonde.yaml

```diff
@@ -26,17 +26,23 @@ observations:
             query: "*/RPID"
           latitude:
             query: "*/CLATH"
+          levelLatitudeDisplacement:
+            query: "*/UARLVB/LATDH"
```

Comment:
what are they using for GNSS-RO, this too has a single value and then individual ones for each level.  the code is designed for a displacement from a single value (as this is pretty specific to a data type)? Or should the converter compute the lat/lon at each vertical point and store this instead.

@emilyhcliu do you have any input on this?